eox_core/api/v1/serializers.py

```diff
@@ -85,4 +85,4 @@ class EdxappCourseEnrollmentQuerySerializer(EdxappCourseEnrollmentSerializer):
     email = serializers.CharField(max_length=255, default=None)
     force_registration = serializers.BooleanField(default=False)
```

Comment:
Can we make it "force" instead of "force_registration"? If we do, does the WP plugin already uses `force_registration` and we require a change there as well?

Reply:
No change needed on the WP plugin

Reply:
cool, then I say, lets do it and merge this